.github/workflows/test.yml

```diff
@@ -0,0 +1,67 @@
+name: Build and Test
+
+on: push
+
+jobs:
+  manager:
+    name: Build and Test
+    
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        # TODO run on Windows as well
```

Comment:
What's preventing from running on Windows?

Reply:
https://github.com/Jigsaw-Code/outline-server/runs/1811847888?check_suite_focus=true

> tar (child): Cannot connect to D: resolve failed
tar: Child returned status 128
tar: Error is not recoverable: exiting now

We fail to build the install script.  We can fix this for sure but my main priority here was to get this out as soon as possible with no reductions in test coverage, and then we can improve later after we've fixed the pressing issue of reducing unit test time.